exercise_008_union_types/README.md

```diff
@@ -0,0 +1,62 @@
+# Union Types
+
+## Background
+
+In the previous exercise, we re-encoded the protocol of the different actors
+in our application from a `sealed trait` based one to one using Dotty
+enumerations. This however, is only a small first step towards a major
+simplification of the way code the exchange of messages between actors.
```

Comment:
`the way code` => `the way we encode`

Reply:
Fixed via [6bbee2d](https://github.com/lunatech-labs/lunatech-scala-2-to-scala3-course/pull/7/commits/6bbee2dd4e120ca9c9b944ef48ea5cc04d10287f)